test/unified_format.py

```diff
@@ -1252,6 +1287,8 @@ def run_entity_operation(self, spec):
             method_name = f"_changeStreamOperation_{opname}"
         elif isinstance(target, NonLazyCursor):
             method_name = f"_cursor_{opname}"
+        elif isinstance(target, CommandCursor):
```

Comment:
What do you think about instead adding CommandCursor to the `_cursor_` elif above?:
```python
        elif isinstance(target, (NonLazyCursor, CommandCursor)):
            method_name = f"_cursor_{opname}"
```

Reply:
Pushed too early sorry, I added CommandCursor as a valid type for all of the `_cursor` test functions.